### DISCUSSION TITLE
Relaxed JSON

### DISCUSSION THREAD

**Discussion by uafcybs** [NeilMacMullen/Textrude#164] [Ideas]
First of all  - thank you for sharing. The great tool. This is almost the perfect solution for my needs that I've been trying to create for almost 10 years. 
The only thing I would add - is a support for [relaxed JSON](http://www.relaxedjson.org/) which would make a model definition much easier and pleasant (IMO). 
I hope I can add it myself even though I'm not a C# guy :) But if you could suggest a quick solution it would be great.

**Comment by NeilMacMullen** [comment]
Thanks for the kind words - always good to hear from people using the textrude :-)  Json deserialisation is performed by [NewtonSoft.Json](https://www.newtonsoft.com/json) so supporting relaxed syntax would be easiest if it's already supported by that library.  I had a quick look and it's not obvious that it is unfortunately.  The alternative would be to find another serialiser library that does support it. 

However, I'd strongly recommend you consider YAML instead - it's considerably easier for humans to write than even "relaxed" json and has better support for comments.  [This](https://phoenixnap.com/blog/what-is-yaml-with-examples) is a good introduction.  If you want to compare directly you can even use Textrude to show you the YAML equivalent of a JSON model file.  Just use a template like this...

```
{{
  model | textrude.to_yaml
}} 
```

![image](https://user-images.githubusercontent.com/9131337/119984375-3bea5500-bfb9-11eb-8006-f4aec18bfd03.png)
